Sources/ComposableArchitecture/SharedState/References/ValueReference.swift

```diff
@@ -328,15 +328,15 @@ extension SharedReader {
 
 private struct LoadError: Error {}
 
-final class ValueReference<Value, Persistence: PersistenceReaderKey<Value>>: Reference, @unchecked
-  Sendable
+final class ValueReference<Value, Persistence: PersistenceReaderKey<Value>>: Reference,
+  @unchecked Sendable
 {
   private let lock = NSRecursiveLock()
   private let persistenceKey: Persistence?
   #if canImport(Combine)
     private let subject: CurrentValueRelay<Value>
   #endif
-  private var subscription: Shared<Value>.Subscription?
+  private var subscription: Shared<Value>.Subscription!
```

Comment:
we'd prefer for this to be a `let` so that we don't have to do `@unchecked Sendable` above, but we need lazy initialization. And to make that clear I went ahead and made it an implicitly unwrapped optional.